saturn-geospatial/recipe-template.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "saturn-geospatial",
+    "description": "A Python image containing libraries specific to geospatial IO, analysis, and visualization. Generally used for CPU instances.",
+    "hardware_type": "cpu",
+    "supports": ["jupyterlab", "dask"]
```

Comment:
did we decide on jupyterlab vs just jupyter?

Reply:
We have not - I proposed it in https://github.com/saturncloud/recipes/pull/36

Reply:
oh I just totally missed that. Ok I think it's fine this way for now.